frontend/src/components/HeaderTitle.vue

```diff
@@ -4,7 +4,7 @@
 
 <style scoped>
 h1 {
-  font-size: 2rem;
+  font-size: 60px;
```

Comment:
Why 60px specifically? As i understand it rem changes things like font size dynamically with the window size?

Reply:
oh lol i didnt mean to change this, i was messing around with the sizes and must have forgotten to change it back